core/domain/exp_services.py

```diff
@@ -807,6 +807,9 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
 
     exploration.version += 1
 
+    # Update summary of changed exploration.
```

Comment:
Woops, the collapsed diff was playing tricks with my eyes, sorry... Okay, I will at the least make changes to the tests that require up to date contributors, which I think at this point is just those calls in ExplorationSummaryGetTests.